lib/logger/lib/logger.ex

```diff
@@ -442,7 +445,12 @@ defmodule Logger do
   def remove_backend(backend, opts \\ []) do
     _ = if opts[:flush], do: flush()
     Logger.Config.remove_backend(backend)
-    Logger.Watcher.unwatch(Logger, Logger.Config.translate_backend(backend))
+    case Logger.Watcher.unwatch(Logger, Logger.Config.translate_backend(backend)) do
+      :ok ->
+        :ok
+      {:error, _} = error ->
+        error
+    end
```

Comment:
Yep, my bad, I accidentally thought this was a case before my changes and wanted to preserve it.